imp_cu_core/include/imp/cu_core/cu_matrix.cuh

```diff
@@ -21,23 +22,29 @@ public:
   __host__ __device__
   ~Matrix() = default;
 
-//  // copy and asignment operator
-//  __host__ __device__
-//  Matrix(const Matrix& other)
-//    : f_(other.f())
-//    , c_(other.c())
-//  {
-//  }
-//  __host__ __device__
-//  Matrix& operator=(const Matrix& other)
-//  {
-//    if  (this != &other)
-//    {
-//      f_ = other.f();
-//      c_ = other.c();
-//    }
-//    return *this;
-//  }
+  __host__
+  Matrix(Eigen::Matrix<Type,_rows,_cols,Eigen::RowMajor> from)
+  {
+    memcpy(data_,from.data(),_rows*_cols*sizeof(Type));
```

Comment:
does Eigen guarantee to be 'unpadded' memory? Or could it happen that there is a stride != cols?